src/slash-bedrock/bin/brl

```diff
@@ -60,14 +60,15 @@ case "${1:-}" in
 	print_help
 	exit_success
 	;;
-"strat" | "list" | "which" | "fetch" | "remove" | "rename" | "copy" | "status" | "enable" | "disable" | "repair" | "hide" | "show" | "alias" | "deref" | "apply" | "update" | "version" | "report" | "tutorial")
-	backend="/bedrock/libexec/brl-${1}"
-	shift
-	exec "${backend}" "${@}"
-	abort "Unable to execute \`${backend}\`"
-	;;
 *)
-	abort "Unrecognized brl subcommand.  See \`${0} --help\`"
+	backend="/bedrock/libexec/brl-${1}"
+	if [ -f $backend ]; then
```

Comment:
```suggestion
	if [ -f "$backend" ]; then
```
This is an edge case, but it's possible.

Reply:
Bedrock shell script style requires such quotes for consistency, even where it's not theoretically needed.  For similar reasons, Bedrock shell script style also requires braces.  Given these, this should be:

```
	if [ -f "${backend}" ]; then
```

Reply:
`-f` was purposefully chosen over `-x` as exec will fail and give you the reason it failed if the file is non-executable. Whereas with `-x` if you had a non-executable file you’ll always get the `unknown subcommand` error message. I can change it up to add a conditional error message if `-x` fails, or simply change it to how you suggest. What would you prefer?

Reply:
Ah, good point!  In this case, I agree `-f` is preferable here.